modules/account-lib/src/coin/hbar/transactionBuilder.ts

```diff
@@ -75,6 +110,47 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
   // endregion
 
   // region Common builder methods
+  /**
+   *  Set the memo
+   *
+   * @param {string} memo A hedera memo
```

Comment:
Is this a hex? Or can be anything?

Reply:
it can be any string up to I think 100 bytes. We should add that limit to the JSDoc. Will work on that alongside the other comments.